dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -185,6 +185,11 @@
     @BatchSize(size = 25)
     private SortedSet<FileFormat> outputFileFormats = new TreeSet<>();
 
+    @OneToMany(fetch = FetchType.LAZY, orphanRemoval = true, cascade = CascadeType.ALL)
```

Comment:
In my document I probably had author in version.  But on 2nd thought, should it be part of version metadata instead (like the existing author)?

Additionally, this could use a comment saying that this is in fact not lazy loaded (despite saying FetchType.LAZY) because of `getAuthor()`

Reply:
> In my document I probably had author in version. But on 2nd thought, should it be part of version metadata instead (like the existing author)?

Good question